docs/finops/README.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FinOps
+description: DevOps-like role in a collaborative approach to optimize cloud costs, ensuring financial accountability, efficiency, and resource management.
+sidebar_position: 9
+---
+
+# FinOps
+
+Understand financial principles relevant to cloud computing, like budgeting, forecasting, cost allocation, and billing.
+
+**Tools offered by cloud providers**:
+
+- AWS Cost Explorer and AWS Budgets
+- Azure Cost Management
+- Google Cloud Billing
+
+The following strategies optimize AWS cloud costs:
+
+**VPC Optimization**:
```

Comment:
The items here should be formated a bit differently like this:

```
- **VPC Optimization**:
  - Manage Data Transfer Costs: NAT gateways incur costs; in most cases, two subnets with 99.9% availability are sufficient. For 99.99% availability, use three subnets with three NAT gateways. Two subnets are generally sufficient for most scenarios.
  - Remove IP v4 addresses that are not needed or associated with Network Interface Card (NIC).
  - Use VPC endpoints to privately connect to supported AWS services and VPC endpoint services, rather than relying on NAT gateways, to reduce costs. Additionally, AWS PrivateLink is often required for other purposes, such as compliance, irrespective of cost considerations.
```